solutions/project/hub-env/securitycontrols.md

```diff
@@ -0,0 +1,46 @@
+# Organization Controls - P1
+
+### AC-2 - (A) ACCOUNT MANAGEMENT
+The organization identifies and selects the following types of information system accounts to support organizational missions/business functions
+
+Address which types of accounts are in use
+* Fortigates VM
+  - One GCP service account is created and attached to both Fortigates VM.
+  - The Fortigates/FortiOS comes with a default local `admin` account.
+  - Federated accounts from Google Cloud Identities are used to allow access to Fortigates serial console.
+
+* Management VM
+  - One GCP service account is created and attached to the `management` VM.
+  - Federated accounts from Google Cloud Identities are used to allow access to the management VM.
+
+# Technical Controls - P1
+
+### AC-3(7) - ACCESS ENFORCEMENT | ROLE-BASED ACCESS CONTROL
+The information system enforces a role-based access control policy over defined subjects and objects and controls access based upon

```

Comment:
It seems that this sentence is incomplete and missing something?

Reply:
@CraigBaltzer @amcmullin please provide your input on this

Reply:
Can the default local 'admin' account be changed, at least in name, or needs to stay?  For the VMs do all federated accounts need access, or select few?

Reply:
@lucstjean-ssc line 19 is an exact copy from this control description in https://www.cyber.gc.ca/en/guidance/annex-3a-security-control-catalogue-itsg-33

Reply:
@amcmullin the default admin account has to be set for bootstrapping the appliance. post configuration may modify it if required.. so out of scope of this package